src/python/grpcio/grpc/experimental/aio/_call.py

```diff
@@ -168,8 +171,10 @@ def __init__(self, cython_call: cygrpc._AioCall, metadata: MetadataType,
         self._response_deserializer = response_deserializer
 
     def __del__(self) -> None:
-        if not self._cython_call.done():
-            self._cancel(_GC_CANCELLATION_DETAILS)
+        # The '_cython_call' object might be destructed before Call object
+        if hasattr(self, '_cython_call'):
```

Comment:
This seems dangerous. What guarantee do we have that `self._cython_call` will not be removed between these two lines on another thread?